laplace/utils/metrics.py

```diff
@@ -0,0 +1,40 @@
+import torch
+from torch.nn import functional as F
+from torchmetrics import Metric
+
+
+class RunningNLLMetric(Metric):
+    """
+    NLL metrics that
+
+    Parameters
+    ----------
+    ignore_index: int, default = -100
+        which class label to ignore when computing the NLL loss
+    """
+    def __init__(self, ignore_index=-100):
+        super().__init__()
+        self.add_state('nll_sum', default=torch.tensor(0.), dist_reduce_fx='sum')
```

Comment:
If we go with my suggestion, we should initialise these tensors on the used device (can be an argument to `__init__`).